### PR TITLE
Ethermon-ERC721 Changes for calculating vote power [ethermon-erc721]

### PR DESCRIPTION
**Pull request by syedMohib44**
Fixes # .

Changes proposed in this pull request:
- Fixed variable swapped EMONA_ADDRESS with EMON_DATA_ADDRESS.
- Somehow .toFixed(0) instead of rounding of number concatinating string causing infite vote power casted that to number.

### CODE REVIEW

File: test/vp.tst.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { getVp } from '../src/utils/vp';
```
Comment (ChaituVR) [inline test/vp.tst.ts:1]: Why this file is renamed? 🤔

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Make sure to revert changes from package-lock.json file. https://github.com/snapshot-labs/snapshot-strategies/pull/813/files should contain only changes for your strategy :)

File: test/vp.tst.ts (hunk 1 above)
Comment (syedMohib44) [reply test/vp.tst.ts:1]: Not sure man just pulled from master

**Review by syedMohib44 — COMMENTED** [review]
@ChaituVR remove invalid vp.tst.ts file and package.json

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**